packages/sdk/src/transaction/transaction.service.ts

```diff
@@ -3,14 +3,15 @@ import BigNumber from "bignumber.js";
 import {
   BigNumber as EthersBigNumber,
   PopulatedTransaction,
-  Wallet,
 } from "ethers";
 import { parseEther } from "ethers/lib/utils";
 import { Config } from "../common/config";
 import { getChainProvider } from "../factory/chain-provider";
 import { gasMultiplier } from "./constants/gas-price-multipliers";
 import { TX_STATUS } from "./constants/transaction-status";
 import TransactionSpeed from "./types/transaction-speed";
+import { JsonRpcSigner } from "@ethersproject/providers";
+import { getWallet } from "src/common/utils";
```

Comment:
this should be import `{ getWallet } from "../common/utils";` like in line 8.

Reply:
Thanks @thomas779 fixing this in another PR